src/ReadingBuddy.java

```diff
@@ -27,6 +27,8 @@ public class ReadingBuddy {
     static boolean isLoggedIn = false;
     static ArrayList<SearchPost> allSearchPosts = new ArrayList<SearchPost>();
     static ArrayList<BookGroup> allBookGroups = new ArrayList<BookGroup>();
+    //NEW SPRINT3
```

Comment:
added to hold all group posts so when a group is ended, the posts for that group can be removed